learn/cloud/analytics.mdx

```diff
@@ -0,0 +1,197 @@
+
+Every time a user clicks on a search result, your application must send a `click` event to the `POST` endpoint of Meilisearch Cloud analytics route:
+
+```sh
+curl \
+  -X POST 'http://edge.meilisearch.com' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer Meilisearch_API_Key'
+  --data-binary '{ 
+    "eventType": "click",
+    "eventName": "search result click",
+    "indexUid": "products",
+    "objectID": "0",
+    "position": 0
+  }'
+```
+
+#### The `click` event object
+
+The `click` event must deliver an object with the following fields:
+
+```json
+{
+  "eventType": "click",
+  "eventName": "search result click",
+  "indexUid": "products",
+  "objectID": "0",
+  "position": 0
+}
+```
+
+- `eventType`: a string indicating this is a `click` event
+- `eventName`: a string describing the event
+- `indexUid`: a string indicating the clicked document's index
+- `objectID`: a string indicating the clicked document's primary key
+- `position`: an integer indicating the clicked document's position in the list of search results
+
+### Configuring conversion rate
+
+To track conversion rate, first identify what should count as a conversion for your application. For example, in a web shop, a conversion might be a user finalizing the checkout process. 
+
+Once you have established what counts as a conversion in your application, configure it to send a `conversion` event to the `POST` endpoint of Meilisearch Cloud analytics route:
+
+```sh
+curl \
+  -X POST 'http://edge.meilisearch.com' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer Meilisearch_API_Key'
+  --data-binary '{ 
+    "eventType": "conversion",
+    "eventName": "product added to cart",
+    "indexUid": "products",
+    "objectID": "0",
+    "position": 0
+  }'
+```
+
+It is not possible to associate multiple `conversion` events with the same search.
+
+#### The `conversion` event object
+
+The `conversion` event must deliver an object with the following fields:
+
+```js
+{
+  "eventType": "conversion",
+  "eventName": "product added to cart",
+  "indexUid": "products",
+  "objectID": "0",
+  "position": 0
+}
+```
+
+- `eventType`: indicates this is a `conversion` event
+- `eventName`: an arbitrary string describing the event
+- `indexUid`: the document's index
+- `objectID`: the document's primary key
+- `position`: the document's position in the list of search results
+
+## Understanding the analytics interface
+
+### Total users and user ids
+
+Indicates the total number of user ids. User ids are set with the `X-MS-USER-ID` HTTP header. If `X-MS-USER-ID` is not present, user ids are generated based on `User-Agent` and `CF-Connecting-IP`.
+
+### Total searches and search IDs
+
+The total number of generated search ids. Meilisearch Cloud generates a new search id for each [user id](#total-users-and-user-ids) after 3 seconds of user inactivity. Multiple queries may be associated with a single search id.
```

Comment:
The old info I have on it is `5` seconds. Need double confirmation 🤓

Reply:
Still 5 seconds https://github.com/meilisearch/meilisearch-edge/blob/main/src/search.js#L84C47-L84C51